src/component/organism/list/InterestTagList.js

```diff
@@ -5,7 +5,7 @@ import {txt} from 'Root/config/textstyle';
 import {btn_w226} from 'Atom/btn/btn_style';
 import {Cross46, Cross52} from 'Atom/icon';
 import AniButton from 'Molecules/button/AniButton';
-import {interestTagList} from 'Organism/style_organism';
+import {interestTagList} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {interestTagList} from 'Organism/style_organism copy';
```